syntax/swift.vim

```diff
@@ -142,6 +145,8 @@ syntax keyword swiftAttributes
       \ @UIApplicationMain
       \ @warn_unused_result
 
+syntax keyword swiftConditionStatement #available
```

Comment:
Yeah I thought about that, but it's not really a pre-processor is it?

But if you're fine with it, I can make that change.